trident/data/line_lists/lines.txt

```diff
@@ -129,7 +129,7 @@ Si II          1808.013000    2.540000e+06    2.490000e-03
 Si II          1309.276000    4.730000e+08    1.130000e-01     Si II* 1309
 Si II          1304.370000    3.040000e+09    1.090000e+00                
 Si II          1264.738000    8.300000e+08    1.940000e-01     Si II* 1265
-Si II          1260.422000    2.950000e+09    1.180000e+01                
+Si II          1260.422000    2.950000e+09    1.180000e+00                
```

Comment:
I'm actually confused why it's including these changes in the PR, as these are already included in the `master` branch.  Maybe when you make the changes I've suggested, github will recognize that the branch you're trying to merge into already has these changes.

Reply:
Honestly I'm not really sure what these are. This was added by @qobilidop as far as I can tell. I certainly have never really looked at the line list.